ngraph/frontend/paddlepaddle/src/op_table.cpp

```diff
@@ -19,9 +19,13 @@
 #include "op/batch_norm.hpp"
 #include "op/relu.hpp"
 #include "op/pool2d.hpp"
-#include "op/elementwise_add.hpp"
+#include "op/elementwise_ops.hpp"
 #include "op/mul.hpp"
 #include "op/scale.hpp"
+#include "op/leakyrelu.hpp"
+#include "op/interp.hpp"
```

Comment:
The name of interp op is different. Please keep file name identical to ops names if possible.

Reply:
Actually we can map several kinds of interpolation ops of PDPD to one function of OV, like -
{"nearest_interp":       "Interpolate"}
{"nearest_interp_v2":  "Interpolate"}
{"bilinear_interp":       "Interpolate"}
{"bilinear_interp_v2":  "Interpolate"}

The things differentiate our Op conversion function are properties. 

 More details of interpolation mapping please refer to ZhangYi3's email sent this week.

Reply:
Since it is M->1 mapping, do you think it is okay to have a unified Op name for this case?

Reply:
The approach is different. It is similar to elementwise ops organization that you provided -- we should have a 1:1 mapping from op to function but in the function itself you can refer to library with shared functions. This table is not a mapping from pdpd ops to ov ops, it is just a way to assign functions to op types. In your example the functions are really different for bilinear vs. nearest, different modes are set for example.

Reply:
For interpolate, yes it is better to group all of them to a single file. Agree.